app/Http/Middleware/isAdmin.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+class isAdmin
```

Comment:
```suggestion
class CheckAdmin
```
sửa lại hết các file liên quan